application/frontend/controllers/AuthenticationController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace frontend\controllers;
+
+use common\models\User;
+use frontend\components\BaseRestController;
+use yii\web\BadRequestHttpException;
+use yii\web\HttpException;
+use yii\web\NotFoundHttpException;
+
+class AuthenticationController extends BaseRestController
+{
+    /**
+     * Authenticates the given user based on his/her password
+     *
+     * @return User upon successful authentication, i.e., "creation".
+     * @throws HttpException
+     */
+    public function actionCreate(): User
+    {
+        $user = User::findOne([
+            'username' => \Yii::$app->request->getBodyParam('username')
+        ]);
+
+        if ($user === null) {
+            throw new NotFoundHttpException();
```

Comment:
yes, thank you, capturing a reminder for that in https://github.com/silinternational/idp-id-broker/commit/830723a2f7970f34f8412cca6f0dba503d9c355f